### DISCUSSION TITLE
Draw child without parent's scale transformation

### DISCUSSION THREAD

**Discussion by mgnslndh** [AvaloniaUI/Avalonia#13018] [Q&A]
Consider the [ZoomBorder ](https://github.com/wieslawsoltes/PanAndZoom)control which applies a pan and zoom tranformation to its child element. How can I setup a child item so that it will be drawn at the correct location but without the applied scale transform? 

Let's assume I have Canvas child to the ZoomBorder so it is possible to pan and zoom that canvas and all its children. But I would like to have one Canvas child control being drawn unscaled while still working properly for panning. I'm not sure how to proceed. Any suggestions?

**Comment by timunie** [comment]
@mgnslndh you probably need to invert the transform matrix. A converter may work. However, calculating the correct position may be not that simple.
